src/grt/src/fastroute/include/FastRoute.h

```diff
@@ -269,7 +269,9 @@ class FastRouteCore
                      const int L,
                      float& slack_th);
   void convertToMazeroute();
-  void updateCongestionHistory(const int upType, bool stopDEC, int& max_adj);
```

Comment:
Any particular reason to change the `updateCongestionHistory` to return an int instead of updating the `max_adj` var in the reference? This is one of the reasons the build is failing: max_adj is not initialized the first time it is used here.

Reply:
I've changed it to return an int instead of updating `max_adj` via reference to make it more explicit that the function is updating `max_adj` in the caller's code. Should I revert it to update via reference?

Reply:
I think it would be better, because it doesn't make much sense to me an update function returning a value. Keeping the reference will also fix the build issue.